great_expectations/jupyter_ux/expectation_explorer.py

```diff
@@ -1911,7 +1911,7 @@ def generate_expectation_suite_editor_widgets(self, data_asset, expectation_suit
         column_count = len(column_names)
         expectation_count = len(expectation_suite["expectations"])
 
-        summary_widget_content = """\
+        summary_widget_content = f"""\
```

Comment:
Runtime change,
ruff is now auto upgrading `.format()` usage to `f-strings`.
https://beta.ruff.rs/docs/rules/f-string/